### DISCUSSION TITLE
About unnecessary settings of requirements-dev.txt, requirements.txt and setup.py

### DISCUSSION THREAD

**Discussion by nsano-rururu** [jertel/elastalert2#491] [Ideas]
share the information. 

**requirements-dev.txt**

sphinx_rtd_theme now supports docutils 0.17, so `docutils<0.17` is not needed.
https://github.com/readthedocs/sphinx_rtd_theme/blob/master/setup.py#L122
Even if you do not write `pluggy>=0.12.0`, the target version is installed, so you do not need to write it.

```
docutils<0.17
pluggy>=0.12.0
```

**requirements.txt**

Since tzlocal2.1, which depends on the installation of apscheduler, is installed, the following settings are not required.
https://github.com/agronholm/apscheduler/blob/3.x/setup.py#L44

```
tzlocal<3.0
```

**setup.py**

Since tzlocal2.1, which depends on the installation of apscheduler, is installed, the following settings are not required.
https://github.com/agronholm/apscheduler/blob/3.x/setup.py#L44

```
'tzlocal<3.0',
```

**Comment by jertel** [comment]
Implemented in #501.
